app/services/PaymentService.scala

```diff
@@ -48,11 +49,18 @@ class PaymentService(val stripeService: StripeService) {
     }
   }
 
+  def identityIdForAccount(purchaserIds: PurchaserIdentifiers) = {
+    purchaserIds.identityId match {
+      case Some(idUser) => idUser.id
+      case None => ""
```

Comment:
I prefer to keep the optionality in the type system if possible, rather than flattening none into an empty string so early